docs/management/schema.rst

```diff
@@ -1,22 +1,11 @@
 .. _schema:
 
-Updating schema
+Catalog schema
 ===============
 
-A specific version of the schema 
-`here <https://github.com/ACCESS-NRI/schema/tree/main/au.org.access-nri/model/output/file-metadata>`_ is 
-downloaded when :code:`access_nri_intake.source` is first imported. This schema is used to validated Intake-ESM 
-datastore entries. Similarly a specific version of the schema 
-`here <https://github.com/ACCESS-NRI/schema/tree/main/au.org.access-nri/model/output/experiment-metadata>`_ is 
-downloaded when :code:`access_nri_intake.catalog` is first imported and this is used to validate 
-intake-dataframe-catalog entries.
-
-Schema can be updated by updating the file(s) at https://github.com/ACCESS-NRI/schema and editing the 
-appropriate :code:`SCHEMA_URL` path(s) in :code:`access_nri_intake.source.__init__` and 
-:code:`access_nri_intake.catalog.__init__`. A hash for the updated schema files is also required (as 
-:code:`SCHEMA_HASH`). The easiest way to update this is to first set :code:`SCHEMA_HASH` to :code:`None`. The 
-updated hash will then be printed to screen when the sub-package is imported and this can be copied and pasted 
-across.
+As of version 0.14, the catalog schema is now a part of the :code:`access_nri_intake_catalog` package, rather
```

Comment:
I think this should be version 0.1.4, not 0.14

Reply:
Correct, I think there might be a couple of those...